jest.config.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
   collectCoverageFrom: [
     'components/**/*.{js,jsx,ts,tsx}',
     'utils/**/*.{js,jsx,ts,tsx}',
+    'data/**/*.{js,jsx,ts,tsx}',
```

Comment:
Does this add the coverage for the tests?

Reply:
Yep, that's right! It means the coverage report that's generated (adding `--coverage` to `yarn test`) will now include line coverage details for files in the `data/` directory